terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -192,8 +192,11 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         site = cls.get_or_throw(Site, "id", kwargs["id"])
         if not user.has_perm(Site.get_perm("change"), site):
             raise cls.not_allowed(MutationTypes.UPDATE)
-        project_id = kwargs.pop("project_id", False)
```

Comment:
comment: had to think about this for a second but couldn't think of any issues with it

Reply:
whoops, this is meant to be on the line after with the `intersection` call